en/docs/reference/mediators/callout-mediator.md

```diff
@@ -117,7 +117,7 @@ The parameters available for configuring the Callout mediator are as follows.
 <tr class="even">
 <td><strong>WS-Security</strong></td>
 <td><div class="content-wrapper">
-<p>If you select the check box, WS-Security is enabled for the Callout mediator. This section would expand as shown below when you select this check box.</p>
+<p>If you select the checkbox, WS-Security is enabled for the Callout mediator. This section would expand as shown below when you select this check box.</p>
```

Comment:
Is there a checkbox now in vs code?

Reply:
It seems that it has been changed to `true` or `false`.

Reply:
Addressed with https://github.com/wso2/docs-mi/pull/538/commits/629f4f40357f33f655d0f4dac75b53a757e7602f